synthdb/synthesis_inputs/luigi_configs/luigi_mouse_Isocortex.cfg

```diff
@@ -0,0 +1,14 @@
+[GetSynthesisInputs]
+url = git@bbpgitlab.epfl.ch:neuromath/synthdb.git
+git_synthesis_input_path = synthdb/insitu_synthesis_inputs/mouse_isocortex
+branch = main
+
+[SynthesisConfig]
+axon_method = synthesis
+
+[RunnerConfig]
+nb_jobs = 1
+
+[BuildMorphsDF]
+neurondb_path = /gpfs/bbp.cscs.ch/project/proj82/home/gevaert/morphology_release/mouse-scaled/scaled_output/06_RepairUnravel-asc/neuronDB.xml
+morphology_dirs = {"morphology_path": "/gpfs/bbp.cscs.ch/project/proj82/home/gevaert/morphology_release/mouse-scaled/scaled_output/06_RepairUnravel-asc","morphology_path_h5": "/gpfs/bbp.cscs.ch/project/proj82/home/gevaert/morphology_release/mouse-scaled/scaled_output/06_RepairUnravel-h5"}
```

Comment:
These URLs are useless, right?

Reply:
I keep them for now, then they'll be on AWS, but never public, this code is not really to be use as public code in fact